docs/whats-new/dotnet-aspire-9.1.md

```diff
@@ -1,18 +1,113 @@
 
 For more information on the official .NET version and .NET Aspire version support, see:
 
 - [.NET support policy](https://dotnet.microsoft.com/platform/support/policy): Definitions for LTS and STS.
 - [.NET Aspire support policy](https://dotnet.microsoft.com/platform/support/policy/aspire): Important unique product life cycle details.
 
--->
\ No newline at end of file
+## Upgrade to 9.1
+
+Moving between minor releases of .NET Aspire is simple:
+
+1. In your app host project file (ie, MyApp.AppHost.csproj), update the Aspire.AppHost.Sdk version to 9.1.0
+`<Sdk Name="Aspire.AppHost.Sdk" Version="9.1.0" />`
+1. Check for any NuGet package updates, either using the NuGet Package Manager in Visual Studio or the "Update NuGet Package" command in VS Code.
+1. Update to the latest .NET Aspire templates by running `dotnet new update` in the command line.
+
+If your project file doesn't have `Aspire.AppHost.Sdk`, you might still be using .NET Aspire 8. To upgrade to 9.1, you can follow [the documentation from last release](../get-started/upgrade-to-aspire-9).
+
+## Dashboard UX and customization
+
+### Resource relationships
+
+The dashboard now has the concept of a "parent" and "child" resource. For example, if you create a Postgres instance with multiple databases, they'll now be nested in the Resource page in the same instance.
+
+![A screenshot of the aspire dashboard showing the Postgres resource with multiple databases nested underneath it](./media/dashboard-parentchild.png)
+
+### Localization overrides
+
+The dashboard defaults to the language that your browser is in. In this release, we added the ability to override that and change the language the dashboard is in separately from the browser language.
+![A screenshot of the aspire dashboard showing the new flyout menu to change language](./media/dashboard-language.png)
+
+### Filtering
+
+You can now filter what you see in the Resource view by type, state, and health status.
+
+### More resource details
+
+When you click on a resource in the dashboard, more data now appears in the details pane, including references, back references, and volumes with their mount types.
+
+![A screenshot of the aspire dashboard with references and back references showing](dashboard-resourcedetails.png)
+
+### CORS support for custom local domains
+
+You can now set the "DOTNET_DASHBOARD_CORS_ALLOWED_ORIGINS" environment variable to allow the dashboard to receive telemetry from other browser apps, such as if you have resources running on custom localhost domains.
+
+### Various UX improvements
+
+There are a bunch of new features in 9.1 that streamline and polish popular tasks, including:
+
+- Start/Stop button now also appears on the console logs page
+- Download console logs to a file
```

Comment:
I think this should be its own section. It's more than a UX improvement.

Reply:
Done!